devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -707,8 +713,16 @@ class WholeBodyDynamicsDevice :  public yarp::dev::DeviceDriver,
       * specified, and a full wrench on the first suitable frame (i.e. frame belonging to the submodel)
       *  is added.
       */
+     //std::string selectedContactFrames{"default"};
      std::vector<std::string> defaultContactFrames;
+     std::vector<std::string> overrideContactFrames;
+     std::vector<std::string> contactWrenchType;
+     std::vector<std::vector<double>> contactWrenchDirection;
+     std::vector<std::vector<double>> contactWrenchPosition;
      std::vector<iDynTree::FrameIndex> subModelIndex2DefaultContact;
+     std::vector<std::vector<iDynTree::FrameIndex>> subModelIndex2OverrideContact;
+     bool overrideContactFramesSelected{false};
+     std::vector<int> subModelVarSize;
```

Comment:
Reading the source code, I realized that this variable is not descriptive enough.

Reply:
How about `nrUnknownsInSubModel`? Done in https://github.com/robotology/whole-body-estimators/pull/48/commits/38c21cdf2a12d33f0fd9a3f736d69353bb89e86e